.github/workflows/code-climate-coverage-aggregation.yml

```diff
@@ -202,19 +208,20 @@ jobs:
           #echo "::set-env name=GIT_BRANCH::master"
         if: env.GIT_BRANCH == 'refs/heads/master'
 
-      # Setup Java Environment
-      - name: Set up JDK 1.8
-        uses: actions/setup-java@v1
-        with:
-          java-version: 1.8
       #- uses: actions/cache@v1
       #  with:
       #    path: ~/.m2/repository
       #    key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
       #    restore-keys: |
       #      ${{ runner.os }}-maven-
 
-      # Run Maven Verify to generate all jacoco reports
+      #Set env again??
+      - name: Set up JDK 17
+        uses: actions/setup-java@v1
+        with:
+          java-version: 17
```

Comment:
We could probably remove this duplicate setup. It was added above so we could build spring-boot-starters (which needed java 17).

Reply:
It actually didn't work. I had to add it twice...

Reply:
strange indeed.